tools/pihole/pihole.md

```diff
@@ -51,7 +32,7 @@ openssl req -new -x509 -keyout server.pem -out server.pem -days 365 -noenc
 
 This pem file now contains a key and a certificate (.crt) file.
 
-Following the above, the `.pem` file needs to be moved to `/etc/lighttpd/ssl/`. The location doesn’t really matter here as long as access is given to the files needed (*sudo chown www-data [pem file]*). "SSL" or "TLS" as the folder name is the most common setup.
+Following the above, the `.pem` file needs to be moved to `/etc/lighttpd/ssl/`. The location doesn't really matter here as long as access is given to the files needed (*sudo chown www-data [pem file]*). "SSL" or "TLS" as the folder name is the most common setup.
```

Comment:
At the very least this guide about adding a self-signed certificate should be updated to reflect the v6 migration.

Maybe it's better to move this to SSL.md as initially planned.